### PR TITLE
feat: Add, remove, and update weapons

### PR DESCRIPTION
**Pull request by Okom1**
New weapons:
* Fuel Rod, REQ 4
* Blaze Rod, REQ 7

Removed weapons:
* Fuelshot, REQ 6 (replaced by Blaze Rod)

Updated weapons:
* Pursuit Hydra: REQ 6 -> 5
* Blaze Hydra: REQ 7 -> 6, damage 0.90 -> 0.70
* Guidedshot: REQ 7 -> 6, ammo 6+3 -> 4+0, damage 1.10 -> 1.00
* Scorpion Tail: Assigned a trait number as we need an ammo cut for it

Other:
* Deleted docs\items\weapons.md as it was the third place where the same weapon info would have to be updated if weapon names or numbers change, and was only used to visualize how many of each base weapon type variant we have.

### CODE REVIEW

File: docs/items/weapon-configs.md [hunk 1]
```diff
@@ -487,16 +487,16 @@ The configurations that make up each of the custom weapons.
 ## [38] Blaze Hydra
```
Comment (ImpliedSkill) [inline docs/items/weapon-configs.md:487]: Remove the blaze hydra

Comment (Okom1) [reply docs/items/weapon-configs.md:487]: Rremoved, and replaced Blaze Hydra slot with the Fuel Rod

File: docs/items/purchases-progressive.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+|                |                    |                                      |                                   |                                        |            Rocket Hog [Vehicle]             |                                        |                                   |                                           |                                      |
+|                |                    |                                      |                                   |                                        |         Campaign Chopper [Vehicle]          |                                        |                                   |                                           |                                      |
+|                |                    |                                      |                                   |                                        |                Pursuit Hydra                |                                        |                                   |                                           |                                      |
```
Comment (ImpliedSkill) [inline docs/items/purchases-progressive.md]: Leave the Pursuit Hydra in REQ cost 6

Comment (Okom1) [reply docs/items/purchases-progressive.md]: Done in latest commit

**Pull request merged**